### ISSUE TITLE
Double Death

### ISSUE THREAD

**Issue by Jadedfey** [LunaPixelStudios/SteamPunk#266]
### Describe the Bug

Everytime I die, I die again immediately on respawn

### Reproduction Steps

1. Find something to fight
2. Lose the fight
3.
...

### Operating System

Windows 10

### Minecraft Version

1.19

### Modloader

Forge

### Modpack Version

23.5

### Optifine

I used Optifine

### Other Mods

No

**Comment by qwek1** [comment]
Could you try installing the pack on another launcher or reinstalling the modpack? Im unable to reproduce this and this is the only report I have gotten of this issue

**Comment by Jadedfey** [comment]
Yeah I can give it a go. I just downloaded through the CursesForge app and use its launcher. On 31 May 2024 7:41 pm, qwek1 ***@***.***> wrote:
Could you try installing the pack on another launcher or reinstalling the modpack? Im unable to reproduce this and this is the only report I have gotten of this issue

**Comment by Josh-Monsoon** [comment]
I have this same issue running from GDlauncher, weird

**Comment by mycroftjr** [comment]
> I have this same issue running from GDlauncher, weird

I also use GDLauncher but have not had this issue. I always equip the Enigmatic Amulet, play on a local server, and didn't take the seven curses.

**Comment by Yogfan14** [comment]
I also have this issue!! Running off of CurseForge launcher. I notice it's considerably more rare if you have a set bed than if you just have a world spawn but yeah

**Comment by Jadedfey** [comment]
> I also have this issue!! Running off of CurseForge launcher. I notice it's considerably more rare if you have a set bed than if you just have a world spawn but yeah

I'm using a fixed bed and unfortunately happens quite a lot.

**Comment by SHXRKIEgg** [comment]
We cant replicate so we dont know how to fix unfortunately, will see what we can do

**Comment by Iris0522** [comment]
My friend and I had the same problem

**Comment by ifeelagood** [comment]
happens to us, too.

**Comment by MarvinMMoura** [comment]
It happened to me too, in a Lan world, when I die, and click to respawn, a second "death" occurs, when I click to respawn, I become invisible, can't interact with anything and can't move. It only fixes when I reenter the world

**Comment by qwek1** [comment]
Seems to happen on both 1.19.2 and 1.20.1 versions of the pack, if anyone is able to figure out the cause of this then please let me know :)

**Comment by NiceClown** [comment]
> Seems to happen on both 1.19.2 and 1.20.1 versions of the pack, if anyone is able to figure out the cause of this then please let me know :)

https://www.youtube.com/watch?v=AOG2_AH29vU
my method for repeating the bug

On the same server (I have it installed via courceforge). If I'm on it alone, then everything is fine, if my friend connects, then this problem begins when the character begins to convulse after several revivals

**Comment by Devon123461** [comment]
It's the enigmatic amulet's extradimensional vessel. Go into the enigmaticlegacy-common config and disable summoning extradimensional vessel on owner's death and it went away for me

**Comment by dawyd69** [comment]
I disable summoning extradimensional vessel on owner's death, and this is what happens. 
https://www.youtube.com/watch?v=yLqR9jTltZI
I even tried to delete the mod but the probeleme is still there

**Comment by Devon123461** [comment]
So now in 1.20.1 v5 it now isn't happening on singleplayer to me anymore it's confusing because it was doing it last night until I disabled that but now even with it enabled it's still not happening. Although on multiplayer it does it every time even with the mod disabled.  But I found if you enable gamerule doImmediateRespawn it's less annoying

**Comment by dawyd69** [comment]
The video was recorded in a lan world where the respawning is not so difficult or anything but on a real server its impossible. 
https://www.youtube.com/watch?v=g5-6eYQp1Eg
And this is with doImmediateRspawn and the summoning extradimensional vessel on owner's death is disabled. Any ideas how to fix?

**Comment by Nevrai** [comment]
I have the same issue. Not LAN.

**Comment by Nevrai** [comment]
I was having the same issue on a server. It happened to multiple people. It’s not a LAN server, and we don’t have OptiFine. We didn’t add any mods or make any config changes. The modpack version is v6 (the latest version as of writing this).

**Comment by GabrielDS26** [comment]
I have the same problem on the latest update, in single player and on LAN, was anyone able to fix this?

**Comment by qwek1** [comment]
issue with the improvable skills 3 mod please report it to them instead
